.buildkite/block.step.yml

```diff
@@ -1,7 +1,12 @@
+agents:
+  queue: 'macos'
```

Comment:
You'll also need to update the queue within buildkite if you haven't done so already